io.github.nokse22.Exhibit.json

```diff
@@ -0,0 +1,71 @@
+{
+    "id" : "io.github.nokse22.Exhibit",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "exhibit",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=dri"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+            {
+	    "name": "python3-f3d",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"f3d\" --no-build-isolation"
+	    ],
+	    "sources": [
+		{
+		    "type": "file",
+		    "url": "https://files.pythonhosted.org/packages/c0/84/56b87aa035fce479e9ec4ac1af1210a273ebede9c2df1b1d06e33f7cc63d/f3d-2.4.0-cp310-cp310-manylinux_2_28_x86_64.whl",
```

Comment:
so who will this work on aarch64?

Also why do you need 4 version of the same arch? Python 3.11 is what's in the runtime.

Reply:
I haven't tested it on aarch64, but the library should work on Arm.

Will fix the packages.